src/components/Header/HeaderSearchBar/HeaderSearchBar.css

```
@@ -5,14 +5,15 @@
     width: 400px;
 }
 
-.search_input {
+.search_product {
     width: 100%;
+    position: relative;
 }
 
 .input {
```

Ще на 1му ПР говорив чому такий підхід поганий. 
Повторю знову: не зважаючи на те, що ці класи ти описуєш в різних файлах НЕ означає, що ці класи стосуються лише компонента до якого ти ці стилі імпортиш. 

Задай ЛЮБОМУ компоненту в твому проекті класс input і побачиш що до нього застосуються ці стилі. 
Є багато способів вирішити цю проблему. Найпростіший, для тебе, на даний момент, це уточнити до чого цей інпут відноситься. 

Бо бандлер (він же збірник) під капотом бере ВСІ css файли та зливає в ОДИН спільний. 
Тому підбирай назву класів так, щоб не ламати верстку в інших місцях. Наприклад:

 ```suggestion
.header_searchBar_input {
```